data/language/en-GB.txt

```diff
@@ -1858,39 +1858,38 @@ STR_2521    :Show staff list
 STR_2522    :Show recent messages
 STR_2523    :Show map
 STR_2524    :Screenshot
-### The following need to be reordered to match SDL_keycode layout.
-STR_2525    :???
-STR_2526    :???
-STR_2527    :???
-STR_2528    :???
-STR_2529    :???
-STR_2530    :???
-STR_2531    :???
-STR_2532    :???
+STR_2525    :<not used anymore>
```

Comment:
Should we just leave these out of the file altogether? I only later noticed there were only very few of these in the file.

Reply:
Might as well.